systemtest/src/main/java/io/strimzi/systemtest/resources/crd/kafkaclients/KafkaAdminClient.java

```diff
@@ -144,6 +148,13 @@ public JobBuilder defaultAdmin() {
         adminLabels.put("app", adminName);
         adminLabels.put(Constants.KAFKA_ADMIN_CLIENT_LABEL_KEY, Constants.KAFKA_ADMIN_CLIENT_LABEL_VALUE);
 
+        PodSpecBuilder podSpecBuilder = new PodSpecBuilder();
+
+        if (Environment.SYSTEM_TEST_STRIMZI_IMAGE_PULL_SECRET != null && !Environment.SYSTEM_TEST_STRIMZI_IMAGE_PULL_SECRET.isEmpty()) {
```

Comment:
So, is this the same secret as used for the operand images? Or a different one?

Reply:
You can specify there any secret you want to create -> and it is also used for the CO and other clients (this was used for the ConnectS2I IIRC)